### ISSUE TITLE
VCM UI V1: Move the user defined dictionary to the `./config` folder and further simplify the UI for adding/setting Vector Collection entries....

### ISSUE THREAD

**Issue by nunoguedelha** [ami-iit/yarp-openmct#115]
The current UI for adding and setting a new Yarp Vector Collection telemetry entry to the visualiser workspace is described in [../doc/add-new-vector-collection-port.md](https://github.com/ami-iit/yarp-openmct/blob/9fdd38cbdb7577d707e3e3d93dcae02e4f5067d2/doc/add-new-vector-collection-port.md).
- We wish to simplify further the set of parameters required to the user, taking advantage of a predefined set of default parameters, required by the OpenMCT plugins for plotting this type of data.
- Only a subset of those default parameters, which are meaningful to the user, shall be overridden through the UI (configuration files in the folder [`../conf`](https://github.com/ami-iit/yarp-openmct/tree/feature/improve-vector-collection-metadata-ui-settings-v1/conf)). The remainders shall be kept away.
- Tht subset of parameters shall be clearly specified in a README.
- In a later PR we shall implement a check to restrict that list.

**Comment by nunoguedelha** [comment]
## Parameters Definition Overview

- Informative template for all the parameters: https://github.com/ami-iit/yarp-openmct/blob/8c0aa66991592e312fc675e7099467713733d07d/openmctStaticServer/plugins/conf/dictionaryBaseTemplate.json#L1-L31

### Default parameters

- Telemetry Metadata default full parameters (`telemetryEntries[i].values` content): https://github.com/ami-iit/yarp-openmct/blob/8c0aa66991592e312fc675e7099467713733d07d/openmctStaticServer/plugins/conf/domainObjTypes.js#L64-L83
- Telemetry Metadata subset of default parameters specific to the metadata type (e.g. `yarpopenmct.veccollectionmap`): https://github.com/ami-iit/yarp-openmct/blob/8c0aa66991592e312fc675e7099467713733d07d/openmctStaticServer/plugins/conf/domainObjTypes.js#L54-L60

### User Defined Parameters

- Telemetry Metadata user UI subset: https://github.com/ami-iit/yarp-openmct/blob/8c0aa66991592e312fc675e7099467713733d07d/conf/dictionaryVectorCollectionsTelemetry.json#L1-L17
- Yarp Port and parser configuration: https://github.com/ami-iit/yarp-openmct/blob/8c0aa66991592e312fc675e7099467713733d07d/conf/servers.json#L81-L89

## Setting Mechanism

```mermaid
graph TD
    A[Init: DEFAULT Full Telemetry Metadata default full parameters] --> B[Override: DEFAULT Type Specific Telemetry Metadata]
    B --> C[Override: USER Telemetry Metadata]
    C --> D(Expand 'range' blocks for all the vectors)
    E[Yarp Port configuration] -->|matching 'walkingController.logger' key| C
    F[telemetry sample] -->|vector keys| D
    D --> G[Final dictionary]
```

**Comment by nunoguedelha** [comment]
Implemented in #113 .

CC @traversaro @RiccardoGrieco @S-Dafarra
